whisper.py

```diff
@@ -32,6 +32,7 @@
 import struct
 import sys
 import time
+import numpy
```

Comment:
Conditional import will be better here, we do not want to bring numpy as default dependency. Like
```python
try:
    import numpy
except ImportError as err:
    pass
```